src/Eyes/Url/UrlStatus.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace BeholderWebClient\Eyes\Url;
+
+use BeholderWebClient\Eyes\Status;
+
+class UrlStatus extends Status {
+
+  const REQUERIMENT_FAIL = 'REQUERIMENT FAIL: php lib curl is not installed.';
+
+  const COULD_NOT_CONNECT = 'Could not connect to url: ';
+  const COULD_NOT_CONNECT_NUMBER = 600;
+
+  const STATUS_CODE_WAS_NOT_EXPECTED = 'The returned status code was not the expected, received: ';
```

Comment:
For example: I bought a gadgets on internet when this package arrived I oponed it, but it's content was not **the** expected. ( Because I was expecting what I bought. )